handbook/engineering/security/index.md

```diff
@@ -4,9 +4,51 @@ We think that security is an enabler for the business. Sourcegraph is committed
 
 ## Goals
 
-Customers know they can confidently and securely use Sourcegraph with their private code.
+### Long term
 
-[This (internal) document](https://docs.google.com/document/d/1HhYfnkC17yCPqqw91H1QAF7q8toOOiNtoOOw06hqrNQ/) details our plan.
+Every organization, is confident they can trust Sourcegraph with the security of their Source code.
+
+### Medium term
+
+The goals below lie on the path to our long-term goals, and represent the next 3-6 months of work.  These goals will be delivered serially, but may be worked on in parallel.
+
+**We share our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph cloud**
+
+  - Problem and rationale: While Sourcegraph cloud is in an good, and improvine security state, we currently do not have the right level of visibility. By starting with a private repository that contains no secrets, we can build and deploy the visibility technologies and processes that allow us to build a holistic view of the environment.  This serves as a proxy for small company repository use.
+  - Planned work:
+    - Run ongoing vulnerability scans of the containers including vulnerability remediation
+    - Deploy centralized security logging
+    - Ship container logs to the logging destination
+    - Begin documenting policies
+  - Definition of success: Frequent container vulnerability scans are being run, and stored for future a audit. Fixes to the containers or their underlying service configurations have made it into the product release cycle.  Logs of new container deployments are available for search in a centralized logging tool.
+
+**We share a private repository containing encrypted secrets**
+
+  - Problem and rationale: Customers will have repositories containing encrypted secrets, using an [ansible vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) or gpg encrypted files. Trusting these repositories to external systems requires the added degree of care and concern, expected in security conscious small and medium enterprise (SME). To support those needs, not only is enhanced visibility into our own systems required on our side, it will be contractually obligated, or at a minimum part of security questionnaires.
+  - Planned work:
```

Comment:
I think we need some sort of RBAC for this. At the very least, we need to reduce the scope of admins from site admins to org admins, so that admins can manage their users without being able to create users in other orgs.

Reply:
@ElizabethStirling: Any substantial changes to our authz and authn systems would currently be driven by the Cloud team as part of their backend infrastructure responsibilities. Please sync with @unknwon to make sure you're on the same page.

Reply:
I did some investigation into this, and I think I'm going to walk back my thoughts on RBAC for now. I'm seeing a way that our current state, or at least a state very close to our current state, could be fine for getting cloud to a state where we're comfortable shipping it to customers.